app/controllers/downloads_controller.rb

```diff
@@ -47,7 +48,7 @@ def load_work
   def remove_downloads
     yield
   ensure
-    Download.remove(@work) unless Rails.env.test?
+    @download.remove
```

Comment:
Do we not want to delete the directory and things like the zip file ?

Reply:
We do, `@download.remove` does that.

Reply:
FileUtils.rm_rf(dir) is not subtle